dvc/commands/plots.py

```diff
@@ -229,6 +229,8 @@ class CmdPlotsTemplates(CmdBase):
         "confusion_normalized",
         "scatter",
         "smooth",
+        "bar_horizontal_sorted",
+        "bar_horizontal",
```

Comment:
@skshetry https://github.com/iterative/dvc/pull/7999 Is to prevent the need for these changes